### DISCUSSION TITLE
Is it allowed to Integrate Amazon SP APIs with google sheet?

### DISCUSSION THREAD

**Discussion by Compliancemode** [amzn/selling-partner-api-models#3934] [General]
We are planning to integrate Amazon SP APIs with google Sheets. Someone said it is not allowed and the developer account can be terminated. Please clarify.

**Comment by chrisskhoury** [comment]
Hello @Compliancemode,

It depends on how you are integrating SP API with Google Sheets. You should not store PII data there.

For some trivial solutions in which you do not store PII data, it should be fine, but I would recommend going over our policies.

You can find them here:

- AUP (Acceptable Use Policy): https://sellercentral.amazon.com/mws/static/policy?documentType=AUP&locale=en_EN
- DPP (Data Protection Policy): https://sellercentral.amazon.com/mws/static/policy?documentType=DPP&locale=en_EN
 
Best,
Christopher El Khoury
Solution Architect, Amazon SP API

**Comment by Compliancemode** [comment]
Hi, thanks for your reply. We do not have pii access. We will not store the
credential as plain text in google sheet. Just wanted to confirm that can
we build a google sheet to store Amazon orders, FBA inventory data using SP
API without PII ?
